test/distributed/_composable/test_composability/test_2d_composability.py

```diff
@@ -279,5 +380,419 @@ def parallelize(_model: Transformer, mesh: DeviceMesh, use_seq_parallel: bool):
         self.assertEqual(loss_no_cp2, loss_cp2)
 
 
+class Test2dParallelIntegration(DTensorTestBase):
+    global LR
+    LR = 3e-5
```

Comment:
not from this PR but this seems... strange to me
Why use a global variable and then set it inside the test class?

Reply:
I make it the same style as the others (lr=...) in the latest commit